client/driver/executor/executor_linux.go

```diff
@@ -116,6 +125,56 @@ func (e *UniversalExecutor) configureCgroups(resources *structs.Resources) error
 	return nil
 }
 
+// Stats reports the resource utilization of the cgroup
+func (e *UniversalExecutor) Stats() (*cstructs.TaskResourceUsage, error) {
+	if !e.command.ResourceLimits {
+		return e.resourceUsagePids()
```

Comment:
I would just throw a comment that we default to the pid based stats if there is no resource isolation